etc/config.pl

```diff
@@ -349,7 +349,7 @@
             google_analytics => 0,
             'inbox_max' => 2000,
             'interests' => 150,
-            'makepoll' => 0,
+            'makepoll' => 1,
```

Comment:
I'm sure you needed to change this for your testing, but it probably shouldn't be included in the pull request.